noir/noir-repo/noir_stdlib/src/scalar_mul.nr

```diff
@@ -26,12 +27,22 @@ impl Add for EmbeddedCurvePoint {
 #[foreign(fixed_base_scalar_mul)]
 // docs:start:fixed_base_embedded_curve
 pub fn fixed_base_embedded_curve(
-    low: Field,
-    high: Field
+    low: Field, // low limb of the scalar
+    high: Field // high limb of the scalar
 ) -> [Field; 2]
 // docs:end:fixed_base_embedded_curve
 {}
 
+// TODO(https://github.com/AztecProtocol/aztec-packages/issues/6061): use a point struct instead of two fields
+#[foreign(variable_base_scalar_mul)]
+pub fn variable_base_embedded_curve(
+    point_x: Field, // x coordinate of a point to multiply the scalar with
+    point_y: Field, // y coordinate of a point to multiply the scalar with
+    scalar_low: Field, // low limb of the scalar
+    scalar_high: Field // high limb of the scalar
+) -> [Field; 2]
+{}
```

Comment:
This will need to be added to the Noir documentation. Can you create an integration test showing the use of this function (similar to others in `test_programs/execution_success`) and then create an entry in `docs/docs/noir/standard_library/scalar.mdx` which demonstrates its use?